src/mlpack/bindings/python/tests/test_python_binding.py

```diff
@@ -140,6 +140,135 @@ def testNumpyMatrixForceCopy(self):
     for j in range(100):
       self.assertEqual(2 * x[j, 2], output['matrix_out'][j, 2])
 
+
+  def testNumpyFContiguousMatrix(self):
+    """
+    The matrix with F_CONTIGUOUS set we pass in, we should get back with the third 
+    dimension doubled and the fifth forgotten.
+    """
+    x = np.array(np.random.rand(100, 5),order = 'F');
```

Comment:
Can you fix the spacing here (and in general in the PR)?  It should be, e.g.,

```
x = np.array(np.random.rand(100, 5), order='F')
```

The keys here are that there are spaces between each argument, but not spaces between keyword arguments.